t/data/tests/tests/soft_failed_test.pm

```diff
@@ -0,0 +1,11 @@
+# Copyright 2019-2021 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+use Mojo::Base 'basetest', -signatures;
+use testapi;
+
+sub run ($) {
```

Comment:
```suggestion
sub run () {
```

The `$` would actually not be signatures but prototypes(?) which we would like to avoid

Reply:
Noting, I had copied it from the samples in the tests.

Reply:
It does need the parameter though, so I rewrote it as `run ($self)`